base/applications/wordpad/lang/hr-HR.rc

```diff
@@ -0,0 +1,258 @@
+        MENUITEM SEPARATOR
+        POPUP "&Dodatno"
+        BEGIN
+            MENUITEM "&Informacije označenoga",             ID_EDIT_SELECTIONINFO
+            MENUITEM "&Format znaka",           ID_EDIT_CHARFORMAT
+            MENUITEM "&Definicija znakovnog formata",           ID_EDIT_DEFCHARFORMAT
+            MENUITEM "&Paragraf formata",           ID_EDIT_PARAFORMAT
+            MENUITEM "Dobij &tekst",                   ID_EDIT_GETTEXT
+        END
+    END
+    POPUP "&Pogled"
+    BEGIN
+        MENUITEM "&Alatna traka",        ID_TOGGLE_TOOLBAR
+        MENUITEM "Traka &formatiranja",     ID_TOGGLE_FORMATBAR
+        MENUITEM "&Ravnalo",          ID_TOGGLE_RULER
+        MENUITEM "&Statusna traka",     ID_TOGGLE_STATUSBAR
+        MENUITEM SEPARATOR
+        MENUITEM "&Opcije...",     ID_VIEWPROPERTIES
+    END
+    POPUP "&Ubaci"
+    BEGIN
+        MENUITEM "&Datum i vrijeme...",  ID_DATETIME
+    END
+    POPUP "&Format"
+    BEGIN
+        MENUITEM "&Font...",                     ID_FONTSETTINGS
+        MENUITEM "&Grafičke oznake",               ID_BULLET
+        MENUITEM "&Paragraf...",                ID_PARAFORMAT
+        MENUITEM "&Kartice...",                     ID_TABSTOPS
+        POPUP "&Pozadina"
+        BEGIN
+            MENUITEM "&Sustav\tCtrl+1",         ID_BACK_1
+            MENUITEM "&Bijelo žuta\tCtrl+2",    ID_BACK_2
+        END
+    END
+    POPUP "&Pomoć"
+    BEGIN
+        MENUITEM "&O Wine WordPad-u",      ID_ABOUT
+    END
+END
+
+IDM_POPUP MENU
+BEGIN
+    POPUP ""
+    BEGIN
+        MENUITEM "Izreži",                ID_EDIT_CUT
+        MENUITEM "&Kopiraj",               ID_EDIT_COPY
+        MENUITEM "&Zalijepi",              ID_EDIT_PASTE
+        MENUITEM SEPARATOR
+        MENUITEM "&Grafičke oznake",      ID_BULLET
+        MENUITEM "&Paragraf...",       ID_PARAFORMAT
+    END
+END
+
+IDM_COLOR_POPUP MENU
+BEGIN
+    POPUP ""
+    BEGIN
+        MENUITEM "Crna",      ID_COLOR_BLACK
+        MENUITEM "Boja kestena",     ID_COLOR_MAROON
+        MENUITEM "Zelena",      ID_COLOR_GREEN
+        MENUITEM "Boja masline",      ID_COLOR_OLIVE
+        MENUITEM "Mornarska boja",       ID_COLOR_NAVY
+        MENUITEM "Ljubičasta",     ID_COLOR_PURPLE
+        MENUITEM "Teal",       ID_COLOR_TEAL
+        MENUITEM "Siva",       ID_COLOR_GRAY
+        MENUITEM "Srebrna",     ID_COLOR_SILVER
+        MENUITEM "Crvena",        ID_COLOR_RED
+        MENUITEM "Lime",       ID_COLOR_LIME
+        MENUITEM "Žuta",     ID_COLOR_YELLOW
+        MENUITEM "Plava",       ID_COLOR_BLUE
+        MENUITEM "Fuchsia",    ID_COLOR_FUCHSIA
+        MENUITEM "Aqua",       ID_COLOR_AQUA
+        MENUITEM "Bijela",      ID_COLOR_WHITE
+        MENUITEM "Automatski",  ID_COLOR_AUTOMATIC
+    END
+END
+
+STRINGTABLE
+BEGIN
+    STRING_ALL_FILES,              "Svi dokumenti (*.*)"
+    STRING_TEXT_FILES_TXT,         "Tekstni dokumenti (*.txt)"
+    STRING_TEXT_FILES_UNICODE_TXT, "Unicode tekstualni dokument (*.txt)"

```

Comment:
What's the difference between "tekstualni dokument" and "tekstni dokument" (that seems to be used everywhere else)?

Reply:
No difference, I will fix it